docs/ref/utils.txt

```diff
@@ -508,6 +508,10 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
         z = person.friends         # does not call
         x is z                     # is True
 
+    .. deprecated:: 4.1
+
+        The ``name`` parameter is deprecated and will be removed in Django 5.0.
```

Comment:
It seems a bit confusing to reference a parameter that doesn't appear in the signature.